onnx/defs/schema.h

```diff
@@ -481,27 +501,58 @@ class OpSchemaRegistry {
         std::cerr << "Schema error: " << e.what() << std::endl;
       }
       auto& m = map();
-      auto& key = op_schema.Name();
+      auto& op_name = op_schema.Name();
+      auto& op_domain = op_schema.domain();
       auto ver = op_schema.SinceVersion();
 
-      if (m[key].count(ver)) {
-        const auto& schema = m[key][ver];
-        std::cerr << "Trying to register schema with name " << key
-                  << " (version " << ver << ") from file " << op_schema.file() << " line "
+      if (m[op_name][op_domain].count(ver)) {
+        const auto& schema = m[op_name][op_domain][ver];
+        std::cerr << "Trying to register schema with name " << op_name
+                  << " (domain: " << op_domain << " version: " << ver
+                  << ") from file " << op_schema.file() << " line "
                   << op_schema.line()
                   << ", but it is already registered from file "
                   << schema.file() << " line " << schema.line();
         abort();
       }
-      m[key].emplace(std::make_pair(ver, op_schema));
+      m[op_name][op_domain].emplace(std::make_pair(ver, op_schema));
     }
   };
 
-  // Return the latest schema for an operator
-  static const OpSchema* Schema(const std::string& key) {
+  // Return the latest schema for an operator in specified domain.
+  // Domain with default value "" means ONNX.
+  static const OpSchema* Schema(
+      const std::string& key,
+      const std::string& domain = "") {
     auto& m = map();
-    if (m.count(key)) {
-      return &m[key].rbegin()->second;
+    if (m.count(key) && m[key].count(domain)) {
+      return &m[key][domain].rbegin()->second;
+    } else {
+      return nullptr;
+    }
+  }
+
+  // Return the schema with biggest version, which is not greater than specified
+  // version in specified domain. Domain with default value "" means ONNX.
+  static const OpSchema* Schema(
+      const std::string& key,
+      const int version,
```

Comment:
I suggest renaming 'version' to 'maxInclusiveVersion' to clarify usage.

Reply:
make sense. fixed.